app/src/layouts/kanban/kanban.vue

```diff
@@ -165,7 +165,7 @@ function openEditGroup(group: Group) {
 function parseAvatar(file: Record<string, any>) {
 	if (!file || !file.type) return;
 	if (file.type.startsWith('image') === false) return;
-	if (file.type.includes('svg')) return;
+	if (!file.type.includes('svg')) return;
 
 	const url = getRootPath() + `assets/${file.id}?modified=${file.modified_on}&width=48&height=48`;
```

Comment:
While we're in here.. Mind updating this to use `key=` with one of the system ones? Direct width/height usage might be disabled

Reply:
Done :)